src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/TestManifests/BadXml/BadXml.xml

```diff
@@ -0,0 +1 @@
+<Build PublishingVersion="3" Name="dotnet-arcade" BuildId="20201016.5" Branch="refs/heads/master" Commit="e7a79ce64f0703c231e6da88b5279dd0bf681b3d" AzureDevOpsAccount="dnceng" AzureDevOpsBranch="refs/heads/master" AzureDevOpsBuildDefinitionId="6" AzureDevOpsBuildId="856354" AzureDevOpsBuildNumber="20201016.5" AzureDevOpsProject="internal" AzureDevOpsRepository="https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-arcade" InitialAssetsLocation="https://dev.azure.com/dnceng/internal/_apis/build/builds/856354/artifacts" IsReleaseOnlyPackageVersion="False" IsStable="False">
```

Comment:
Yup. These files are all so small that it would be waaaay better if they were just strings inside the tests.

That might mean the file reading part of the code needs to get pulled apart from all the logic, but that's goodness anyway.